package.json

```diff
@@ -16,6 +16,9 @@
     "build:emulate-ci": "CI=true CIRCLECI=true npm run build:production",
     "test:unit": "jest --config='__tests__/config/jest.unit.config.js' --testLocationInResults",
     "test:e2e": "npm run db:seed && jest --config='__tests__/config/jest.e2e.config.js' --testLocationInResults --runInBand",
+    "///": "When https://github.com/facebook/jest/issues/6470 is merged we'll be able to do this properly, but for now it's just so that flaky tests don't break CI too often",
+    "test:unit:ci": "npm run test:unit -- --ci || npm run test:unit -- --ci || npm run test:unit -- --ci",
```

Comment:
LOL at the `||`. 😂

 Have you thought about adding something to the style guide that specifies a format for `todo` comments? I like having comments like the above labeled in some uniform way so that people can grep the database for stuff that should be fixed e.g. `//TODO(Emil) When this PR gets merged change this so we can do it properly`.